docs/TestData.fsx

```diff
@@ -35,6 +35,13 @@ This dictionary is used to find an arbitrary instance for properties that have a
 based on the argument's type.
 
 Arbitrary instances have some helper functions in `Arb`.
+
+## Shrinkers
+
+When writing custom shrinkers, keep in mind that:
+
+- The sequence of shrinks generated by the built-in shrinkers may be infinite.
```

Comment:
This is not accurate. The built-in shrinkers all terminate for all inputs. If they do not, it's a bug. 

Shrinking may take a long time though, but typically in pathological, constructed cases (such as a test that only fails for a particularly long input and for nothing else, and so all shrinks are tried without success)

A common problem with writing custom shrinkers is that they fail to terminate, typically because a shrinker returns a value that was already tried.